components/Banner/index.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from "react";
+import styled from "styled-components";
+
+export default function Banner({ bannerStatus, setShowBanner }) {
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setShowBanner(false); // Hide the banner after 3 seconds (adjust as needed)
```

Comment:
Bad UX, better to allow user to see this banner and close it by clicking on some button :)

Reply:
And btw, what is it for?